src/include/homestore/btree/btree.ipp

```diff
@@ -280,8 +273,7 @@ out:
 #ifndef NDEBUG
     check_lock_debug();
 #endif
-    if (ret != btree_status_t::success && ret != btree_status_t::has_more &&
-        ret != btree_status_t::fast_path_not_possible) {
+    if ((ret != btree_status_t::success) && (ret != btree_status_t::has_more)) {
```

Comment:
Can you refresh me the reason we don't need 'fast_path_not_needed', I may have missed the reason for it.